grommet/grommet
# Remove BoxSnowpack

#### What does this PR do?
This PR removes the BoxSnowpack test component that was added in #5202 because it was only needed for testing and never meant to be released.

#### Where should the reviewer start?

#### What testing has been done on this PR?

#### How should this be manually tested?
Check that files added in #5202 are removed in this PR (note: DateInput-test.js.snap was changed in 5202 but it shouldn't be reverted because the snapshots changes were due to the month change April->May and not related to BoxSnowpack).

#### Any background context you want to provide?

#### What are the relevant issues?

#### Screenshots (if appropriate)

#### Do the grommet docs need to be updated?
No

#### Should this PR be mentioned in the release notes?
No

#### Is this change backwards compatible or is it a breaking change?
Backwards compatible
